src/kernel/expr.cpp

```diff
@@ -295,6 +295,12 @@ bool is_default_var_name(name const & n) {
     return n == *g_default_name;
 }
 
+extern "C" uint8 lean_expr_isHave(object * e);
+bool let_nonDep_core(expr const & e) {
```

Comment:
Similarly to `lean_expr_isHave` making more sense as `lean_expr_is_have`, I think `let_nonDep_core` makes more sense as `let_non_dep_core`, and similarly for other uses of `non_dep`. The camel case really sticks out like a sore thumb (hump?) in the C++ code.

Reply:
Given that nondependent is a word, I went with `let_nondep` and `let_nondep_core`. Any objection to that? (I might switch `nonDep` to `nondep` in Lean later.)

Reply:
Yeah that seems fine. There's already an `is_nondep_elim` in `lcnf.cpp`.